src/components/MenuItem.vue

```diff
@@ -21,6 +21,7 @@
 </template>
 
 <script>
+import vp from '@/lib/viewport'
```

Comment:
Are you actually still scaling the element smaller/bigger than 128px? If not, this could be removed.